README.md

```diff
@@ -1,13 +1,3 @@
 # Google Student Training in Engineering Program
```

Comment:
Let's try to make each PR self-contained and avoid including the same changes in other PRs. Can you please delete the changes here and in the other PRs that you have made in a previous PR?

Reply:
protip: I usually use a similar workflow to this when working on a PR that depends on another: https://softwareengineering.stackexchange.com/questions/351727/working-on-a-branch-with-a-dependence-on-another-branch-that-is-being-reviewed/351790#351790